### PR TITLE
Implement Github Markdown CSS

### LINKED ISSUE #1643: Fix Markdown header sizing
Content within Markdown is only ever shown beneath a top-level header. We should make sure that Markdown content is sized appropriately.

Not this:

image

### PR DESCRIPTION
**Pull request by jbrooksuk**
Closes #1643 

---

@joecohens I think this is a good start, probably needs some overrides for margins though. Thoughts?

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
-        "jquery-serialize-object": "~2.5",
-        "jquery-sparkline": "~2.1",
-        "livestampjs": "~1.1",
-        "lodash": "~4.5.1",
-        "messenger": "~1.4",
-        "moment": "~2.11.2",
-        "Sortable": "~1.4",
-        "sweetalert": "~1.1"
-    }
+  "name": "Cachet",
```
Comment (GrahamCampbell) [inline bower.json]: 4 spaces gone wrong here

Comment (jbrooksuk) [inline bower.json]: This is what happens when you run `bower install --save package`. I'm sick of fixing the bloody thing.

Comment (joecohens) [inline bower.json]: Yeah bower does this each time you install a new dep

**Comment by jbrooksuk** [comment]
It'll do for now.

**Comment by joecohens** [comment]
:heart_eyes:

**Pull request merged**